### PR TITLE
Feature/TRI-1350 Update tavern tests to new aspects and testdataset

### PR DESCRIPTION
**Pull request by ds-alexander-bulgakov**

### CODE REVIEW

File: .github/workflows/tavern.yml [hunk 1]
```diff
@@ -54,7 +59,8 @@ jobs:
           KEYCLOAK_HOST: ${{ secrets.KEYCLOAK_OAUTH2_CLIENT_TOKEN_URI }}
           KEYCLOAK_CLIENT_ID: ${{ secrets.KEYCLOAK_OAUTH2_CLIENT_ID }}
           KEYCLOAK_CLIENT_SECRET: ${{ secrets.KEYCLOAK_OAUTH2_CLIENT_SECRET }}
-          GLOBAL_ASSET_ID: ${{ github.event.inputs.global-asset-id || 'urn:uuid:513d7be8-e7e4-49f4-a22b-8cd31317e454' }}
+          GLOBAL_ASSET_ID_AsPlanned: ${{ github.event.inputs.global-asset-id-asPlanned || 'urn:uuid:0733946c-59c6-41ae-9570-cb43a6e4c79e' }}
+          GLOBAL_ASSET_ID_AsBuild: ${{ github.event.inputs.global-asset-id-asBuild || 'urn:uuid:7940e0cc-7814-41eb-8b04-d984a325deec' }}
```
Comment (ds-jhartmann) [inline .github/workflows/tavern.yml:63]: Please set the env vars as all caps e.g. GLOBAL_ASSET_ID_AS_PLANNED, GLOBAL_ASSET_ID_AS_BUILD

File: local/testing/api-tests/irs-api-tests.tavern.yaml
Comment (ds-jhartmann) [inline local/testing/api-tests/irs-api-tests.tavern.yaml:1]: Please set the env vars as all caps e.g. GLOBAL_ASSET_ID_AS_PLANNED, GLOBAL_ASSET_ID_AS_BUILD

File: irs-load-tests/src/test/java/org/eclipse/tractusx/irs/IRSLoadTestSimulation.java [hunk 2]
```diff
@@ -31,7 +31,7 @@ public class IRSLoadTestSimulation extends Simulation {
                                                .acceptHeader("*/*");
 
         Map<CharSequence, String> headers_1 = new HashMap<>();
-        headers_1.put("Authorization", "Bearer #{access_token}");
+        headers_1.put("Authorization", "Bearer {access_token}");
```
Comment (ds-jhartmann) [inline irs-load-tests/src/test/java/org/eclipse/tractusx/irs/IRSLoadTestSimulation.java:34]: Why was this changed?

**Review by ds-jhartmann — CHANGES_REQUESTED** [review]

File: irs-load-tests/src/test/java/org/eclipse/tractusx/irs/IRSLoadTestSimulation.java (hunk 2 above)
Comment (ds-alexander-bulgakov) [reply irs-load-tests/src/test/java/org/eclipse/tractusx/irs/IRSLoadTestSimulation.java]: Probably accidentally. I corrected it!

**Review by ds-jhartmann — APPROVED** [review]

**Pull request merged**